torch/distributions/categorical.py

```diff
@@ -59,7 +66,7 @@ def __init__(self, probs=None, logits=None, validate_args=None):
             if probs.dim() < 1:
                 raise ValueError("`probs` parameter must be at least one-dimensional.")
             self.probs = probs / probs.sum(-1, keepdim=True)
-        else:
+        if logits is not None:  # Note: 'if is None' instead of 'else' to help mypy
```

Comment:
Isn't that a better way to satisfy linter?
```suggestion
        else:
            assert logits is not None
```

Reply:
AS per suggestion I changed to this style in several spots